dotCMS/src/integration-test/java/com/dotmarketing/startup/runonce/Task05370ChangeContainerPathToAbsoluteTest.java

```diff
@@ -0,0 +1,354 @@
+                                "</div>" +
+                                "<div class=\"clear\"></div>" +
+                                "<h2>Container: Default</h2>" +
+                                "<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>" +
+                            "</span>" +
+                            "<div style=\"display: none;\" title=\"container_69b3d24d-7e80-4be6-b04a-d352d16493ee\" id=\"splitBody0_div_69b3d24d-7e80-4be6-b04a-d352d16493ee_1562770692396\">" +
+                                "#parseContainer('%s/application/containers/default/','1')\n" +
+                            "</div>" +
+                        "</div>" +
+                    "</div>" +
+                "</div>" +
+                "<div id=\"ft-template\">" +
+                    "<h1>Footer</h1>" +
+                "</div>" +
+            "</div>\n";
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to Test: {@link Task05370ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with relative path container
+     * Should: Should turn it into a Absolute Path, using the template's host
+     */
+    @Test
+    public void whenTemplateLayoutHasRelativePathShouldTurnIntoAAbsolutePath() throws IOException, DotDataException, DotSecurityException {
+
+        final String layout = String.format(jsonDrawBody, "");
+        final String testBody = String.format(body, "");
+        final Host host = new SiteDataGen().nextPersisted();
+
+        checkTemplateLayout(layout);
+        final Contentlet theme = new ThemeDataGen().nextPersisted();
+        final Template template = new TemplateDataGen()
+                .title("template_test_" + System.currentTimeMillis())
+                .theme(theme)
+                .drawedBody(layout)
+                .body(testBody)
+                .host(host)
+                .nextPersisted();
+
+
+        final Task05370ChangeContainerPathToAbsolute task05370ChangeContainerPathToAbsolute =
+                new Task05370ChangeContainerPathToAbsolute();
+
+        task05370ChangeContainerPathToAbsolute.executeUpgrade();
+
+        checkTemplateFromDataBase(host, template);
+    }
+
+    /**
+     * Method to Test: {@link Task05370ChangeContainerPathToAbsolute#executeUpgrade()}
+     * When: Exists A TemplateLayout with relative path container in the drawed_body fields and the body field is null
+     * Should: Should turn it into a Absolute Path, using the template's host
+     */
+    @Test
+    public void whenTemplateLayoutHasRelativePathButBodyIsNullShouldTurnIntoAAbsolutePath() throws IOException, DotDataException, DotSecurityException {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5990100)